src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1229,7 +1229,7 @@ class AddItemsPage extends Component {
   submitRequest(lineItems) {
     const nonEmptyLineItems = _.filter(lineItems, val => !_.isEmpty(val) && val.product);
     this.saveRequisitionItems(nonEmptyLineItems)
-      .then(() => this.transitionToNextStep('REQUESTED'));
+      .then(() => this.transitionToNextStep('PENDING_APPROVAL'));
```

Comment:
see the comment above - either you bring back those lines in the `PENDING_APPROVAL` case, or we should should check (or actually we might do it in both places) here, if a location supports approval workflow - if it does, send `PENDING_APPROVAL`, otherwise after your refactor, I guess we should send `VERIFYING` directly, instead of `REQUESTED`.